errors.yaml

```diff
@@ -0,0 +1,51 @@
+---
+errors:
+  - code: create_index
+    description: "Error relative to index creation, check out our guide on [index creation](link.com)"
+  - code: existing_index
+    description: "An index with this name already exists, check out our guide on [index creation](link.com)"
+  - code: invalid_index_uid
+    description: "The provided index formatting is wrong, check out our guide on [index creation](link.com)"
+  - code: open_index
+    description: "An error occured while trying to open the requested index, ..."
```

Comment:
```suggestion
    description: "An error occurred while trying to open the requested index, ..."
```

Reply:
Don't have any idea to complete the `...`

Reply:
Also, I don't think it is clear what "opening" an index means